src/fights.ts

```diff
@@ -96,6 +97,7 @@ import {
 } from "./outfit";
 import { withStash } from "./clan";
 import { withChoice, withChoices } from "libram/dist/property";
+import { log } from ".";
```

Comment:
can you put log in its own file? Don't like importing stuff from index

Reply:
That seems reasonable. I'll move globalOptions as well